test/udfs/test_gaussian_blur.py

```diff
@@ -0,0 +1,28 @@
+import os
+import PIL
+
+import torch
+
+from eva.udfs.gaussian_blur import GaussianBlur
+
+from test.udfs.test_utils import load_image, get_udf_data_path
+
+
+def test_gaussian_blur_PIL():
+    img_path = os.path.join(get_udf_data_path(), "dog.jpeg")
+    img = PIL.Image.fromarray(load_image(img_path))
+
+    blur = GaussianBlur(kernel_size=3, sigma=0.5)
+
+    blurred_img = blur.transform(img)
+    assert type(img) == type(blurred_img)
```

Comment:
Is there a better way to check if the image is blurred?

Reply:
I did not want to couple the test to the exact implementation of torch's gaussian blur.
That is not the point.
Instead I wanted to make sure the EVA UDF handles the frame correctly and the output is something we expect.